verilog/tools/lint/README.md

```diff
@@ -124,17 +124,30 @@ The `--rules` flag allows to enable/disable rules as well as pass configuration
 to rules that accept them. It accepts a comma-separated list
 [rule names][lint-rule-list]. If prefixed with a `-` (minus), the rule is
 disabled. No prefix or a '+' (plus) prefix enables the rule. An optional
-configuration can be passed after an `=` assignment.
+configuration can be passed after an `=` assignment. Each name/value is
+
+So rule configurations with parameters looks like this
+```
+  --rules=[+-]rule-name=<param>:<paramvalue>;<param2>:<value>,[+-]next-rule...
```

Comment:
Possible suggestion: `;` is a shell token, so the flag may need to be single- or double-quote protected, as you have an in example below.

Reply:
Good idea. Uupdated.